app/views/gardens/_form.html.erb

```diff
@@ -11,5 +11,5 @@
  <%= f.input :picture %>
  <%= f.input :picture_cache, as: :hidden %>
 
- <%= f.button :submit, "Edit!", class: "btn btn-success btn-lg"%>
+ <%= f.button :submit, "List my garden!", class: "btn btn-success btn-lg"%>
```

Comment:
This shouldn't have text because we use the same form for new and edit. Simple for will render the corresponding text.